src/transformers/models/open_llama/configuration_open_llama.py

```diff
@@ -116,7 +116,10 @@ def __init__(
         self.initializer_range = initializer_range
         self.rms_norm_eps = rms_norm_eps
         self.use_cache = use_cache
-        self.use_memorry_efficient_attention = use_memorry_efficient_attention
+        if "use_memorry_efficient_attention" in kwargs:
+            self.use_memory_efficient_attention = kwargs["use_memorry_efficient_attention"]
```

Comment:
```suggestion
            self.use_memory_efficient_attention = kwargs.pop("use_memorry_efficient_attention")
```
If you leave it here, it will still set a `self.use_memorry_efficient_attention`